src/test/java/io/lettuce/core/commands/StreamCommandIntegrationTests.java

```diff
@@ -49,6 +30,23 @@
 import io.lettuce.core.protocol.CommandArgs;
 import io.lettuce.test.LettuceExtension;
 import io.lettuce.test.condition.EnabledOnCommand;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
nit: why it keeps moving imports ? is it IDE thing ?

Reply:
Yes, but I agree it is a bit too much. I will disable it.

Reply:
The thing is - when enabled it automatically removes unused imports, and sorts all imports alphabetically and I liked that a lot, but it also makes the change unnecessarily bigger. Not sure which is the better approach.